src/openvic-simulation/interface/GUI.hpp

```diff
@@ -40,15 +40,14 @@ namespace OpenVic::GUI {
 	class Scene : public Named<UIManager const&> {
 		friend std::unique_ptr<Scene> std::make_unique<Scene>();
 
-		NamedInstanceRegistry<Element, UIManager const&> elements;
+		NamedInstanceRegistry<Element, UIManager const&> IDENTIFIER_REGISTRY(scene_element)
 
 	protected:
-		Scene();
+		Scene() = default;
 
 		bool _fill_key_map(NodeTools::key_map_t& key_map, UIManager const& ui_manager) override;
 
 	public:
-		Scene(Scene&&) = default;
```

Comment:
Couple small things in this file:
- can you add back the default move constructor here,
- and add semicolons to the ends of the `scene_element` and `window_element` registry declarations,
- and add back the newline back at the end of the file.